pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -25,6 +26,29 @@ const ( // Rootfs type enum
 	ErofsRootfs                          // Create a plain erofs rootfs
 )
 
+func (r *RootfsType) UnmarshalJSON(data []byte) error {
+	var s string
+	if err := json.Unmarshal(data, &s); err != nil {
+		return err
+	}
+	switch s {
+	case "ext4", "":
```

Comment:
I think this should be `squashfs-ext4` to make sure there is no confusion thinking it's plain ext4 or that arbitrary filesystem types can be used. This is also consistent with what you can pass to lorax's `--rootfs-type`

Reply:
Thanks, yeah, that makes sense. Updated.